README.rst

```diff
@@ -158,6 +171,41 @@ with the following fields:
     }
 
 
+If the file is gzipped by the server, an ``original`` key is added in the ``attachment``
+key, containing the file info **before** it's gzipped. The ``attachment`` keys are
+in that case referring to the gzipped file:
+
+
+.. code-block:: json
+
+    {
+        "data": {
+            "attachment": {
+                "filename": "IMG_20150219_174559.jpg.gz",
+                "hash": "hPME6i9avCf/LFaznYr+sHtwQEX7mXYHSu+vgtygpM8=",
+                "location": "http://cdn.service.org/files/ffa9c7b9-7561-406b-b7f9-e00ac94644ff.jpg.gz",
+                "mimetype": "application/x-gzip",
+                "size": 14818,
+                "original": {
```

Comment:
In the current implementation of the Fennec client ``original`` is not inside the `attachment` key object but outside at the same level as the `attachment` key.